index.html

```diff
@@ -576,355 +533,6 @@ <h3>
-                "resource-timing-2#dom-performanceresourcetiming-redirectend"><code>
-                redirectEnd</code></a>, <a data-link-for=
-                "PerformanceNavigationTiming">redirectCount</a>,
-                <a data-link-for="PerformanceNavigationTiming">type</a>,
-                <a data-cite=
-                "resource-timing-2#dom-performanceresourcetiming-nexthopprotocol">
-                <code>nextHopProtocol</code></a>, <a data-link-for=
-                "PerformanceNavigationTiming">unloadEventStart</a> and
-                <a data-link-for=
-                "PerformanceNavigationTiming">unloadEventEnd</a>. Set
-                <a data-cite=
-                "resource-timing-2#dom-performanceresourcetiming-nexthopprotocol">
-                <code>nextHopProtocol</code></a> to the empty {{DOMString}}.
-              </li>
-              <li>Return to <a href="#fetch-start-step">fetch-start-step</a>
-              with the new resource.
-              </li>
-            </ol>
-          </li>
-          <li>Record the time as <a data-link-for=
-          "PerformanceNavigationTiming">domInteractive</a> immediately before
-          the user agent sets the <a data-cite=
-          "HTML/dom.html#current-document-readiness">current document
-          readiness</a> to "interactive".
-          </li>
-          <li>Record the time as <a data-link-for=
-          "PerformanceNavigationTiming">domContentLoadedEventStart</a>
-          immediately before the user agent fires the <a data-cite=
-          "HTML/parsing.html#the-end">DOMContentLoaded event</a> at the
-          document.
-          </li>
-          <li>Record the time as <a data-link-for=
-          "PerformanceNavigationTiming">domContentLoadedEventEnd</a>
-          immediately after the <a data-cite=
-          "HTML/parsing.html#the-end">DOMContentLoaded event</a> completes.
-          </li>
-          <li>Record the time as <a data-link-for=
-          "PerformanceNavigationTiming">domComplete</a> immediately before the
-          user agent sets the <a data-cite=
-          "HTML/dom.html#current-document-readiness">current document
-          readiness</a> to "complete".
-          </li>
-          <li>Record the time as <a data-link-for=
-          "PerformanceNavigationTiming">loadEventStart</a> immediately before
-          the user agent fires the load event.
-          </li>
-          <li>Record the time as <a data-link-for=
-          "PerformanceNavigationTiming">loadEventEnd</a> immediately after the
-          user agent completes the load event.
-          </li>
-          <li>Set the <a>duration</a> to a {{DOMHighResTimeStamp}} equal to the
-          difference between <a data-link-for=
-          "PerformanceNavigationTiming">loadEventEnd</a> and {{startTime}},
-          respectively.
-          </li>
-          <li>
-            <a data-cite=
-            'performance-timeline-2#dfn-queue-a-performanceentry'>Queue</a> the
```

Comment:
Same for queueing the entry - I didn't see it in the HTML PR

Reply:
Yes, didn't get to it yet :)

Reply:
now it is